src/applications/mhv-supply-reordering/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useRef } from 'react';
```

Comment:
Note: This file is as generated by the app generator. We will need to changes this file as needed when we develop the confirmation page.